### PR TITLE
add webapp monitor example

### PR DESCRIPTION
**Pull request by edgarriba**
Image to be added later

Screenshot from 2023-10-04 15-40-28

### CODE REVIEW

**Review by ethanrublee — COMMENTED** [review]
That's cool. I think this one needs an example of how to add it as an app that can be launched in the launcher. But love it!

**Review by guilhermedemouraa — COMMENTED** [review]
I couldn't build the website locally. Got this error:
Screenshot from 2023-10-04 10-14-50

File: website/docs/examples/monitor_app/README.md [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+
+```bash
+pip3 install -r requirements.txt
+```
+
+### 3. Install the frontend dependencies
+
+Recommendation: Use [**Node Version Manager**](
+  <https://www.digitalocean.com/community/tutorials/how-to-install-node-js-on-ubuntu-20-04>
+  #option-3-installing-node-using-the-node-version-manager) to install Node.js 18.x.
```
Comment (guilhermedemouraa) [inline website/docs/examples/monitor_app/README.md:252]: I had some problems here.
When simply doing:
```bash
sudo apt install nodejs
sudo apt install npm
```
Apparently, the node version installed (v10.19.0) is not compatible with the `vite` version.
I had to do:
```bash
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
nvm install --lts
nvm use --lts
```

I am not sure if you had the same problem or if we should worry about adding more steps here, but I thought I'd let you know

**Review by guilhermedemouraa — APPROVED** [review]
Tested :ok_hand:

**Review by guilhermedemouraa — APPROVED** [review]
This is a really nice example. I think we should add a brief explanation of how apps work and everything required to create an app. Consider adding the following:

### Building an App: Understanding the Basics

Every modern app typically consists of two main parts: the `backend` and the `frontend`. Let's dive into what each of these components does and how they interact with each other.

#### Backend
The backend is like the brain of your app. It processes data, makes calculations, communicates with databases, and performs all the logical operations. When you hear terms like "server," "API," or "database," they're usually related to the backend.

In our example, we're using FastAPI to build our backend. FastAPI is a modern, high-performance web framework for building APIs. Coupled with the farm-ng brain services, our backend will fetch and serve data efficiently and securely to the frontend.

Key Points:

- Handles data processing, storage, and retrieval.
- Communicates with other services and databases.
- Secures data and ensures only authorized users can access it.


#### Frontend
The frontend is the part of the app users see and interact with. Think of it as the face of your app. It includes everything that you can touch, click, or interact with: buttons, images, text inputs, animations, and more.

For our frontend, we're using React. React is a popular JavaScript library for building user interfaces. It allows developers to create responsive and interactive UI components easily.

Key Points:

- Displays data fetched from the backend.
- Interacts with users, capturing their inputs and preferences.
- Updates in real-time, ensuring users always see the latest data.

#### How They Work Together
- A user interacts with the frontend (e.g., clicks a button to fetch information).
- The frontend sends a request to the backend, asking for specific data.
- The backend processes the request, fetches the data (from databases, other services, or the farm-ng brain services in our case).
- Once the data is retrieved, the backend sends it back to the frontend.
- The frontend then displays this data to the user in a readable and interactive manner.

**Comment by edgarriba** [comment]
@guilhermedemouraa next time commit directly your suggestions!

**Pull request merged**